ProcessMaker/Listeners/BpmnSubscriber.php

```diff
@@ -150,13 +152,55 @@ public function onIntermediateCatchEventActivated(IntermediateCatchEventInterfac
         }
     }
 
+    public function updateDataWithFlowTransition($gateway, $flow, $instance)
+    {
+        // Exit job if flow doesn't have set a config attribute
+        if (empty($flow->getProperties()['config'])) {
+            return;
+        }
+
+        // Exit if config is not a valid json
+        if (empty(json_decode($flow->getProperties()['config']))) {
+            Log::error('Flow config attribut is not a valid json');
+            return;
+        }
+
+        // Exit if no variable or expression is set
+        $config = json_decode($flow->getProperties()['config'], true);
+        if ( empty($config['update_data'])
+            || empty($config['update_data']['variable'])
+            || empty($config['update_data']['expression'])
+        ) {
+            return;
+        }
+
+        try {
+            $variable = $config['update_data']['variable'];
+            $expression = $config['update_data']['expression'];
+
+            $formalExp = new FormalExpression();
+            $formalExp->setLanguage('FEEL');
+            $formalExp->setBody($expression);
+            $expressionResult = $formalExp($instance->data);
+            $data = array_merge($instance->data, [$variable => $expressionResult]);
+            $instance->data = $data;
+            $instance->getDataStore()->setData($instance->data);
+            $instance->saveOrFail();
+        } catch (\Exception $e) {
+            Log::error('The expression used in the flow generated and error: ', $e->getMessage());
```

Comment:
Consider throw an error and log the message in the request log
```
$instance->logError();
```

Reply:
done